blazor/diagram/connectors/interactions.md

```diff
@@ -219,6 +217,182 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ![Dragging Connector End Point in Blazor Diagram](../images/blazor-diagram-drag-connector-end-point.gif)
 
+## How to route the connectors
+
+The connectors may overlap with adjacent nodes when a node is positioned in such a way that it encounters the connector. This overlap can reduce the clarity of the connector's path. To prevent this, the Routing process can be utilized.
```

Comment:
The connectors may overlap with adjacent nodes when a node is positioned so that it encounters the connector. This overlap can reduce the clarity of the connector's path. To prevent this, the Routing process can be utilized.